tests/prefix_caching/test_disable_sliding_window.py

```diff
@@ -26,6 +27,11 @@
 ]
 
 
+@pytest.mark.skipif(
+    torch.cuda.get_device_capability() < (8, 0),
```

Comment:
Instead of skipping, we should do:

```python
dtype="auto"
if torch.cuda.get_device_capability() < (8, 0):
    dtype="half"
vllm_disabled_model = LLM(model, disable_sliding_window=True, dtype=dtype)
```